cirq-rigetti/cirq_rigetti/service_test.py

```diff
@@ -8,4 +8,4 @@ def test_get_rigetti_qcs_service():
     """test that get_rigetti_qcs_service can initialize a `RigettiQCSService`
     through `pyquil.get_qc`."""
     service = get_rigetti_qcs_service('9q-square', as_qvm=True, noisy=False)
-    assert service._quantum_computer.name == '9q-square-qvm' # pragma: no cover
+    assert service._quantum_computer.name == '9q-square-qvm'  # pragma: no cover
```

Comment:
This is already part of a test, so this probably just means that the test wasn't called?

Reply:
For tests marked with `rigetti_integration` you may need to pass the `--rigetti-integration` option to pytest.
The CI coverage check already does so [here](https://github.com/quantumlib/Cirq/blob/ef4c1f33c097872c3bdbf7a1e1e9e6a80d97ddbf/.github/workflows/ci.yml#L226); this line should not cause CI complaints about coverage, ie, please remove `no cover`.